index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./main.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav class="nav-bar">
+            <h2>Regiss M.</h2>
+            <div><img src="./Images/Icon-Menu.svg" alt="menu"></div>
+        </nav>
+        <div class="header-title">
+            <h1>Hey There. I’m Regiss</h1>
+            <h3>I'am software developer</h3>
+            <p>I can help you build a product. feature or website look through some of my work and experience! if you
+                like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            <div class="logo-plat">
+                <ul>
+                    <li><a href="https://github.com/Regiss05" target=”_blank”><img src="./Images/github.svg"
+                                alt="github" /></a></li>
+                    <li><a href="https://www.linkedin.com/in/regiss-mukubiza-1bab841b3/" target=”_blank”><img
+                                src="./Images/linkedin.svg" alt="linkedin" /></a></li>
+                    <li><img src="./Images/Tert.svg" alt="Tert" /></li>
+                    <li><a href="https://twitter.com/regissmukubiza" target=”_blank”><img src="./Images/twitter.svg"
+                                alt="twitter" /></a></li>
+                    <li><a href="https://medium.com/@regiss05" target=”_blank”><img src="./Images/Medium.svg"
+                                alt="Medium" /></a></li>
+                </ul>
+            </div>
+        </div>
+    </header>
+    <div class="container">
```

Comment:
- [ ] Kindly change the div tag to a section tag as requested by the previous reviewer, please make all appropriate changes requested by previous reviewer 👍🏿